doc/source/dev/build-perf/simd_pythran.rst

```diff
@@ -0,0 +1,170 @@
+Enabling SIMD for Pythran
+=========================
+
```

Comment:
How about starting with something like:

_The SciPy wheels on PyPI do not use Pythran's SIMD support, because they need to be compatible with old CPUs. However, if you build SciPy from source you can enabled building with SIMD instructions supported by your CPU, which may yield better performance. ...._